### PR TITLE
DDF-3079 Catalog UI documentation

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR updates Catalog UI documentation.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@ricklarsen @troymohl 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
@adimka 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@millerw8
@jlcsmith 
#### How should this be tested? (List steps with links to updated documentation)
Build / Visually inspect changes
#### What are the relevant tickets?
[DDF-2866](https://codice.atlassian.net/browse/DDF-2866)
[DDF-2820](https://codice.atlassian.net/browse/DDF-2820)
[DDF-2894](https://codice.atlassian.net/browse/DDF-2894)
[DDF-2848](https://codice.atlassian.net/browse/DDF-2848)
#### Checklist:
- [X] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc [hunk 1]
```diff
@@ -137,10 +138,12 @@ Text searches are by default case insensitive, but case sensitive searches are a
 +
 *Between*: set a beginning and end time to search between.
 
-[[_spatial_search_details]]Spatial Search Details:: Search by latitude/longitude or the USNG using a line, polygon, point-radius, or bounding box.
+[[_spatial_search_details]]Spatial Search Details:: Search by latitude/longitude, USNG, or UTM using a line, polygon, point-radius, or bounding box.
 
 [[_type_search_details]]Type Search Details:: Search for specific content types.
 
+[[_sort_search_details]]Sort Parameters:: Specify which Attribute and order to search sort the Query results.
```
Comment (troymohl) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: Maybe a little better wording:
+[[_sort_search_details]]Sort Parameters:: Specify which Attribute(s) and it's sort order for Query result sorting.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: or even `Specify sort attribute(s) and query results ordering`..?

Comment (bdeining) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: changed

**Comment by bdeining** [comment]
build now

File: distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc [hunk 2]
```diff
@@ -107,14 +107,15 @@ View, edit, and add searches from the *Searches* tab.
 * *Duplicate*: Create a copy of this search as a starting point
 * *Search Archived*: Execute the search, but specifically for archived results.
 * *Search Historical*: Execute the search, but specifically for historical results.
+* *Submit Feedback*: Submit Query feedback to a configurable email address
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: nit: period at the end

Comment (bdeining) [inline distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc]: fixed

**Comment by clockard** [comment]
build now

**Pull request merged**